flake.nix

```diff
@@ -131,7 +131,11 @@
             targets = [ target ];
           };
 
-      pre-commit-builder = { rust ? mkRust { } }: pre-commit-hooks.lib.${system}.run {
+      # A note on check_format: the way we invoke rustfmt here works locally but fails on CI.
+      # Since the formatting is checked on CI anyway - as part of the rustfmt check - we
+      # disable rustfmt in the pre-commit hook when running checks, but enable it when
+      # running in a dev shell.
+      pre-commit-builder = { rust ? mkRust { }, check_format ? false }: pre-commit-hooks.lib.${system}.run {
```

Comment:
Nix convention is camelCase, so it should be `checkFormat`

Reply:
Oops, thank you!